README.md

```diff
@@ -1,15 +1,104 @@
+[![StyleCI](https://styleci.io/repos/:style_ci_id/shield)](https://styleci.io/repos/:style_ci_id)
+[![SensioLabsInsight](https://img.shields.io/sensiolabs/i/:sensio_labs_id.svg?style=flat-square)](https://insight.sensiolabs.com/projects/:sensio_labs_id)
+[![Quality Score](https://img.shields.io/scrutinizer/g/laravel-notification-channels/workplace.svg?style=flat-square)](https://scrutinizer-ci.com/g/laravel-notification-channels/workplace)
+[![Code Coverage](https://img.shields.io/scrutinizer/coverage/g/laravel-notification-channels/workplace/master.svg?style=flat-square)](https://scrutinizer-ci.com/g/laravel-notification-channels/workplace/?branch=master)
+[![Total Downloads](https://img.shields.io/packagist/dt/laravel-notification-channels/workplace.svg?style=flat-square)](https://packagist.org/packages/laravel-notification-channels/workplace)
 
-### I'm working on a new channel
-Please create an issue for it if it does not already exist, then PR you code for review.
+This package makes it easy to send notifications using [Workplace](https://work.facebook.com) with Laravel 5.5+ and 6.0
 
-## Workflow for new channels
+## Contents
 
-1) Clone the [skeleteon repo](https://github.com/laravel-notification-channels/skeleton)
-2) Use find/replace to replace all of the placeholders with the correct values (package name, author name, email, etc)
-3) Submit a new PR agaisnt this repo for review
+- [Installation](#installation)
+- [Usage](#usage)
+	- [Available Message methods](#available-message-methods)
+- [Changelog](#changelog)
+- [Testing](#testing)
+- [Security](#security)
+- [Contributing](#contributing)
+- [Credits](#credits)
+- [License](#license)
 
-Take a look at our [FAQ](http://laravel-notification-channels.com/) to see our small list of rules, to provide top-notch notification channels.
+
+## Installation
+
+You can install the package via composer:
+
+``` bash
+composer require laravel-notification-channels/workplace
+```
+
+## Usage
+
+You can now use the channel in your `via()` method inside the Notification class.
+
+``` php
+use NotificationChannels\Workplace\WorkplaceChannel;
+use NotificationChannels\Workplace\WorkplaceMessage;
+use Illuminate\Notifications\Notification;
+
+class WorkplaceNotification extends Notification
+{
+    public function via($notifiable)
+    {
+        return [WorkplaceChannel::class];
+    }
+
+    public function toWorkplace($notifiable)
+    {
+        return new WorkplaceMessage('# Laravel Notification Channels are awesome!');
+    }
+}
+```
+
+You also need to add the `routeNotificationForWorkplace()` method in your notifiable model:
+
+``` php
+class TestNotifiable
+{
+    use \Illuminate\Notifications\Notifiable;
+
+    public function routeNotificationForWorkplace()
+    {
+        return 'https://graph.facebook.com/<groupId>/feed?access_token=<access_token>';
```

Comment:
I don't think this is the best way to route notifications.

Instead, we should built a `WorkplaceClient` class that interacts with the API.

Add a workplace services to the `services.php` config:
```php
'workplace' => [
    'access_token' => env('WORKPLACE_TOKEN')
]
```

And then pass that to the client's constructor.

Then, when routing the notifications the method should return just the `groupId` to post to

Reply:
Hey @atymic really sorry for the huge delay.
I totally agree with you that it makes sense to have a `WorkplaceClient` class that interacts with the API. I have thought about it and looked into existing implementations, but ended up with these approach because of this scenario:
Every bot you have on Workplace has a specific `access_token`, and then on workplaces, you give them permissions to post to whatever groups you think is best. The problem with having a single `access_token` is that I can't use different bots from my Workplace and have to stick with a single one.
This is also the way Laravel has the Slack Notifications configured, although that doesn't mean this is the best/correct way to do it.
Cheers

Reply:
No worries, thanks for the explanation. 
Happy for it to stay as in then!

Reply:
Ping me once the other comments are resolved and i'll re-review